HIP/hip-873.md

```diff
@@ -0,0 +1,103 @@
+---
+hip: 873
+title: Adding Decimal Information to Token Balance Calls in the Hedera REST API
+author: Houdini NFT (@HoudiniNFT), Ty Smith (@ty-swirldslabs)
+requested-by: Atlantis
+type: Standards Track
+category: Mirror
+needs-council-approval: Yes
+status: Review
+created: 2024-01-30
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/874
```

Comment:
This discussion doesn't exist and a PR has taken that number

Reply:
Updated to the discussion of this PR. Thank you!